hoomd/md/methods/methods.py

```diff
@@ -79,6 +79,11 @@ class NVT(Method):
 
         `NVT` integrates rotational degrees of freedom.
 
+    Note:
+        The NVT integrator is reversible. In addition to negating velocities
```

Comment:
This may not be true, after further discussion with @joaander in Slack. The acceleration applied to the thermostat depends on the difference between the instantaneous temperature and the set point, which may not work correctly if reversed. But if NVT isn't reversible, is the NPT integrator reversible? Does the NPT thermostat work the same way or not?